tests/Oryx.RuntimeImage.Tests/Node/NodeRuntimeImageCanRunWhenAppInsightsModuleNotFound.cs

```diff
@@ -86,11 +86,11 @@ public async Task GeneratesScript_CanRun_AppInsights_NotConfigured(string nodeVe
             var volume = DockerVolume.CreateMirror(hostDir);
             var appDir = volume.ContainerDir;
             var imageName = string.Concat("oryxdevmcr.azurecr.io/public/oryx/node-", nodeVersion);
-            var aIEnabled = "APPLICATIONINSIGHTSAGENT_EXTENSION_ENABLED";
+            var aIEnabled = "ApplicationInsightsAgent_EXTENSION_VERSION";
             int containerDebugPort = 8080;
 
             var script = new ShellScriptBuilder()
-                .AddCommand($"export {aIEnabled}=TRUE")
+                .AddCommand($"export {aIEnabled}=disabled")
```

Comment:
I think this test will fail. You can make this `~2` to simulate the actual value.

Reply:
well im only setting up the extension_version env variable , I didn't set the ai_key .. so I'm expecting my app will run but app insights wont be configured. but I will double check my theory .. Right now I'm building my images locally .. I will test all the tests once the images are built .. in the mean time I created the PR for more visibility towards other people :) .. but thanks for pointing out